src/NServiceBus.Core/DelayedDelivery/DelayedDeliveryOptionExtensions.cs

```diff
@@ -22,6 +22,20 @@ public static void DelayDeliveryWith(this SendOptions options, TimeSpan delay)
 
             options.GetExtensions().AddDeliveryConstraint(new DelayDeliveryWith(delay));
         }
+
+        /// <summary>
+        /// Returns the configured delivery delay by using <see cref="DelayDeliveryWith"/>.
+        /// </summary>
+        /// <param name="options">The options being extended.</param>
+        /// <returns>The configured <see cref="TimeSpan"/> or <c>null</c>.</returns>
+        public static TimeSpan? GetDeliveryDelay(this SendOptions options)
```

Comment:
We talked quickly about that idea because we have other getters returning strings where null also means no value. We stayed with the "null == no value" since all setters ensure users can't set null values.